build.xml

```diff
@@ -200,6 +202,16 @@
         </exec>
     </target>
 
+    <target name="project-base-vendor-check" description="Check there exists no vendor directory in project-base/ as that would break the autoloader" hidden="true">
+        <if>
+            <available file="${path.root}/vendor" type="dir" property="dir.Exists"/>
+            <then>
+                <echo level="error" message="Composer dependencies are installed in ${path.root} directory which would break the monorepo autoloader. Remove ${path.root}/vendor before re-running the command,"/>
```

Comment:
```suggestion
                <echo level="error" message="Composer dependencies are installed in ${path.root} directory which would break the monorepo autoloader. Remove ${path.root}/vendor before re-running the command."/>
```